databases/models/User.js

```diff
@@ -83,6 +83,11 @@ module.exports = (sequelize, DataTypes) => {
       is_backdoor_user: {
         type: DataTypes.BOOLEAN,
         defaultValue: false
+      },
+      verified_status: {
+        type: DataTypes.ENUM('VERIFIED', 'UNVERIFIED'),
+        allowNull: false,
+        defaultValue: 'VERIFIED'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `verified_status` field is a good idea for tracking whether a user's profile is verified or not. However, it might be better to consider using more generic terms for the ENUM values, such as 'YES' and 'NO', or 'TRUE' and 'FALSE'. This would make the field more reusable in case you need to track verification status for other features in the future.
```diff
- type: DataTypes.ENUM('VERIFIED', 'UNVERIFIED'),
+ type: DataTypes.ENUM('YES', 'NO'),
```
Also, ensure that there are corresponding changes in the application logic to handle this new field appropriately.


<!-- This is an auto-generated comment by OSS CodeRabbit -->